File: 2576.py
def odd(num):
	return num % 2 == 1

odd_li = []

for i in range(7):
	num = int(input())
	if odd(num):
		odd_li.append(num)

if odd_li == []:
	print("-1")
else:
	print(sum(odd_li))
	print(min(odd_li))

File: 2576.cpp
#include <stdio.h>

int main(){
    int c = 7;
    int sum = 0;
    int min = 99;
    while(c--){
        int n;
        scanf("%d", &n);
        if(n % 2 == 0) continue;
        sum += n;
        if(n < min) min = n;
    }
    if(sum == 0) printf("-1\n");
    else printf("%d\n%d\n", sum, min);
    return 0;
}